carAPI/src/main/java/com/everis/entity/Car.java

```diff
@@ -0,0 +1,68 @@
+package com.everis.entity;
+
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+
+public class Car {
+	private long id;
+	private String brand;
+	private LocalDateTime registration;
+	private String country;
+	private LocalDateTime created_at;
```

Comment:
Field attributes are main written in [camelCase](https://pt.wikipedia.org/wiki/CamelCase) and not [snake_case](https://en.wikipedia.org/wiki/Snake_case). Please check it.